tasks-by-users/Naufil-Shemle/01-e-commerce-website/index.html

```diff
@@ -0,0 +1,410 @@
+		<div class="row">
+			<div class="col-2">
+				<img src="images/exclusive.png" class="offer-img" alt="Exclusive Product Image">
+			</div>
+			<div class="col-2">
+				<p>Exclusively Available on RedStore</p>
+				<h1>Smart Band 4</h1>
+				<small>The Mi Band 4 featrue a 39.9% larger (than Mi Band 3) AMOLED color full-touch display with adjustable brightness, so everything is clear as can be.</small>
+				<a href="" class="btn">Buy Now &#8594;</a>
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<!-- -----------------Testimonial---------------- -->
+
+
+<div class="testimonial">
+	<div class="small-container">
+		<div class="row">
+			<div class="col-3">
+				<i class="bi bi-chat-left-quote-fill"></i>
+				<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla suscipit cumque tempora totam a natus, magni dolores repellendus quisquam ut!</p>
+				<div class="rating">
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star"></i>
+				</div>
+				<img src="images/user-1.png" alt="User Image">
+				<h3>Sean Parker</h3>
+			</div>
+
+			<div class="col-3">
+				<i class="bi bi-chat-left-quote-fill"></i>
+				<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla suscipit cumque tempora totam a natus, magni dolores repellendus quisquam ut!</p>
+				<div class="rating">
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star"></i>
+				</div>
+				<img src="images/user-2.png" alt="User Image">
+				<h3>Mike Smith</h3>
+			</div>
+
+			<div class="col-3">
+				<i class="bi bi-chat-left-quote-fill"></i>
+				<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla suscipit cumque tempora totam a natus, magni dolores repellendus quisquam ut!</p>
+				<div class="rating">
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star-fill"></i>
+					<i class="bi bi-star"></i>
+				</div>
+				<img src="images/user-3.png" alt="User Image">
+				<h3>Mabel Joe</h3>
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<!-- -----------------Brandl---------------- -->
+
+
+<div class="brand">
+	<div class="small-container">
+		<div class="row">
+			<div class="col-5">
+				<img src="images/logo-godrej.png" alt="godrej logo">
+			</div>
+			<div class="col-5">
+				<img src="images/logo-oppo.png" alt="oppo logo">
+			</div>
+			<div class="col-5">
+				<img src="images/logo-coca-cola.png" alt="coca-cola logo">
+			</div>
+			<div class="col-5">
+				<img src="images/logo-paypal.png" alt="paypal logo">
+			</div>
+			<div class="col-5">
+				<img src="images/logo-philips.png" alt="philips logo">
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<!-- -----------------Brand---------------- -->
+
+
+<div class="footer">
```

Comment:
Use `footer` tag

Reply:
I will keep this point from next time, this time I was just following the tutorial.